torba/torba/server/script.py

```diff
@@ -51,13 +51,13 @@ def __init__(self, name, enumList):
             if isinstance(x, tuple):
                 x, i = x
             if not isinstance(x, str):
-                raise EnumError("enum name {} not a string".format(x))
+                raise EnumError(f"enum name {x} not a string")
             if not isinstance(i, int):
-                raise EnumError("enum value {} not an integer".format(i))
+                raise EnumError(f"enum value {i} not an integer")
             if x in uniqueNames:
-                raise EnumError("enum name {} not unique".format(x))
+                raise EnumError(f"enum name {x} not unique")
             if i in uniqueValues:
-                raise EnumError("enum value {} not unique".format(x))
+                raise EnumError(f"enum value {x} not unique")
```

Comment:
Should that be `{i}` instead of `{x}`?

Reply:
I think so, looks like a bug in the original too. Shall I change it here?

Reply:
yes